source/features/useful-forks.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'dom-chef';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import {RepoForkedIcon} from '@primer/octicons-react';
+
+import features from '.';
+import {getRepo} from '../github-helpers';
+import looseParseInt from '../helpers/loose-parse-int';
+
+async function init(): Promise<void | false> {
+	const forkCount = looseParseInt((await elementReady('.social-count[href$="/network/members"]', {waitForChildren: false}))!);
+	if (forkCount === 0) {
+		return false;
+	}
+
+	const downloadUrl = new URL('https://useful-forks.github.io');
+	downloadUrl.searchParams.set('repo', getRepo()!.nameWithOwner);
+
+	const selector = pageDetect.isRepoForksList() ? '#network' : '#repo-content-pjax-container h2';
+	(await elementReady(selector, {waitForChildren: false}))!.prepend(
```

Comment:
Too many punctuation characters on one line make it a little hard to parse. Also if the suggestion above applies, the selector can be inlined in here
```suggestion
	const container = await elementReady(selector, {waitForChildren: false});
	container!.prepend(
```